### PR TITLE
add: user provisioning for Okta with SCIM

### PR DESCRIPTION
**Pull request by staceysalamon-aiven**
## Describe your changes

Add article on SCIM user provisioning to meet Okta requirements for integration.

## Checklist

- [X] The first paragraph of the page is on one line.
- [X] The other lines have a line break at 90 characters.
- [X] I checked the output.
- [X] I applied the [style guide](../styleguide.md).
- [X] My links start with `/docs/`.

### CODE REVIEW

**Comment by staceysalamon-aiven** [comment]
Preview: https://staceys-okta-scim-doc-951.aiven-docs.pages.dev/docs/platform/howto/okta-user-provisioning-with-scim

File: docs/platform/howto/okta-user-provisioning-with-scim.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+1. Click **Test API Credentials** to confirm the connection is working
+   and save the configuration.
+1. Click **Sign On**.
+1. In the **Credentials Details** section, select **Email** as the
+   **Application username format**.
+1. Click **Save**.
+
+You can assign Okta users to the Aiven application. They get an invite to your Aiven
+organization.
+
```
Comment (acutaia) [inline docs/platform/howto/okta-user-provisioning-with-scim.md:40]: I am not sure about this @m-novikov

File: docs/platform/howto/okta-user-provisioning-with-scim.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+1. Select **Enable API Integration**.
+1. In the **API Token field**, paste the **Access token** from the Aiven Console.
+1. Click **Test API Credentials** to confirm the connection is working
+   and save the configuration.
+1. Click **Sign On**.
+1. In the **Credentials Details** section, select **Email** as the
+   **Application username format**.
+1. Click **Save**.
+
+You can assign Okta users to the Aiven application. They get an invite to your Aiven
```
Comment (m-novikov) [inline docs/platform/howto/okta-user-provisioning-with-scim.md:38]: Users provisioned by SCIM will be managed users and will be members of organization without going through invite process.

File: docs/platform/howto/okta-user-provisioning-with-scim.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+---
+title: Configure user provisioning for Okta
+limited: true
+---
+
+You can automate user provisioning with Okta through System for Cross-domain Identity Management (SCIM).
+This means you can manage your users and their profiles in one place, Okta, and push
+those changes to the Aiven platform.
```
Comment (ArthurFlag) [inline docs/platform/howto/okta-user-provisioning-with-scim.md:8]: This should be on one line please :)

File: docs/platform/howto/okta-user-provisioning-with-scim.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+Aiven’s integration with Okta supports these features:
+
+- **Push new users**  - Users created in Okta are automatically created as managed users
```
Comment (ArthurFlag) [inline docs/platform/howto/okta-user-provisioning-with-scim.md:12]: We typically use a colon for such definition enumerations, from what I can tell? Could we harmonize?

**Review by acutaia — APPROVED** [review]
LGTM

**Review by ArthurFlag — APPROVED** [review]

**Pull request merged**